### PR TITLE
CTA update

### PR DESCRIPTION
**Pull request by bekahjaned**
Added gradient to primary and secondary buttons on Home page
Screen Shot 2020-11-29 at 9 56 56 AM
Screen Shot 2020-11-29 at 9 57 02 AM

### CODE REVIEW

**Review by MarkCortejos — CHANGES_REQUESTED** [review]
just some merging fragments that need to be cleaned up. thanks!

File: styles/styles.css [hunk 1]
```diff
@@ -243,6 +243,48 @@ body {
 }
 
 .button-primary {
+<<<<<<< HEAD
```
Comment (MarkCortejos) [inline styles/styles.css:246]: please delete

File: styles/styles.css [hunk 2]
```diff
@@ -243,6 +243,48 @@ body {
...
+
+.button-secondary:hover {
+  border: none;
+  background-image: linear-gradient(312deg, #0e0e0e, #9b0227, #e9002d, #9f001e, #0e0e0e);
+  background-position: 100% 50%;
+  -webkit-box-shadow: 0 4px 15px 0 rgba(233, 0, 45, 0.6);
+          box-shadow: 0 4px 15px 0 rgba(233, 0, 45, 0.6);
+  -webkit-transition: all 0.4s ease-in-out;
+  transition: all 0.4s ease-in-out;
+=======
```
Comment (MarkCortejos) [inline styles/styles.css:287]: please delete

File: styles/styles.css [hunk 3]
```diff
@@ -316,6 +358,7 @@ body {
 .button-secondary::after {
   bottom: 0;
   right: 0;
+>>>>>>> 40d63d8415701a0dec66428b3f024816df832a3b
```
Comment (MarkCortejos) [inline styles/styles.css:361]: please delete

**Comment by bekahjaned** [comment]
Cleaned up merging fragments.

**Review by MarkCortejos — APPROVED** [review]

**Pull request merged**